BurgerGov.cs

```diff
@@ -0,0 +1,159 @@
+using System.Numerics;
+using Neo;
+using Neo.SmartContract;
+using Neo.SmartContract.Framework;
+using Neo.SmartContract.Framework.Native;
+using Neo.SmartContract.Framework.Services;
+using Neo.SmartContract.Framework.Attributes;
+
+
+namespace NeoBurger
+{
+    [ManifestExtra("Author", "NEOBURGER")]
+    [ManifestExtra("Email", "developer@neo.org")]
+    [ManifestExtra("Description", "NeoBurger Governance Token")]
+    [SupportedStandards("NEP-17")]
+    [ContractPermission("*", "*")]
+    public class BurgerGov : Nep17Token
```

Comment:
the name can be `NeoBurgerGovernanceToken` and the filename can be `NeoBurgerGovernanceToken.cs`